vorestation.dme

```diff
@@ -4191,6 +4194,8 @@
 #include "maps\expedition_vr\beach\submaps\mountains.dm"
 #include "maps\expedition_vr\beach\submaps\mountains_areas.dm"
 #include "maps\gateway_archive_vr\blackmarketpackers.dm"
+#include "maps\offmap_vr\om_ships\abductor.dm"
+#include "maps\offmap_vr\om_ships\lunaship.dm"

```

Comment:
These should probably not be automatically loaded as well.

Reply:
Bit confused how they ended up within my #include since I did not touch those files at all.

c'est la vie. Those caused the merge conflict earlier as well, which I misunderstood and added them.
Tried removing them now.